cypress/e2e/facility_spec/FacilityHomepage.cy.ts

```diff
@@ -136,6 +138,22 @@ describe("Facility Homepage Function", () => {
     facilityHome.verifyLiveMonitorUrl();
   });
 
+  it("Verify sidebar collapse and expand functionality", () => {
+    facilityHome.toggleSidebar();
+    facilityHome.verifyIconsVisible();
+    facilityHome.verifyTextVisible();
```

Comment:
- it does not verify that each icon has a respective text next to it.

Reply:
@nihal467 sir I have made changes for this & sorry for those syntax & duplicate errors I tried to fix merge conflicts directly & made those mistakes